### PR TITLE
Create contact form

### PR DESCRIPTION
**Pull request by eamador1**
Since the last review, we worked on the following changes:

- [ ] Add a contact form
- [ ] Test linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -179,37 +179,47 @@ <h2 class="project-header">About</h2>
...
-    </div>
-      <div class="email-section">
-        <label class="email-label">Your email</label><br>
-        <input type="text" class="email" name="email" placeholder="anastasiia@gm...">
-    </div>
-    <div class="message-section">
-        <label class="message-label">Your message</label><br>
-        <input type="text" class="message" placeholder="Enter your message here...">
-    </div>
+      <form action="forms/" method="post">
```
Comment (bobb-Rob) [inline index.html:190]: - [ ] Based on the project requirement, the submission of this form should be handled by [formspree](https://formspree.io/html). This means that you will create an account with Formspree, get a `form_id` then, copy your unique form url for use in the form element as shown below 👇🏾 

```html
  <form action="https://formspree.io/f/form_id" method="post"> 
```

In the action property above, replace the form_id text there with your actual form_id value obtained from your Formspree account. 

```html
  <form action="https://formspree.io/f/23ead6" method="post"> <!--  the form_id was replaced with my formspree generated Id which is 23ead6 -->
```

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @eamador1, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by eamador1** [comment]
Since last review, the following changes were implemented:

- [ ] Give text-boxes a 4px border-radius
- [ ] Change email type to email
- [ ] Change border attribute to none
- [ ] Implement form submission handled by spree

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @eamador1 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -515,6 +551,8 @@ a:hover {
   width: auto;
   justify-content: space-between;
   padding-bottom: 20px;
+  padding-left: 5px;
+  padding-right: 10px;
 }
```
Comment (eamador1) [inline style.css:556]: Great job with the coding. Good indentation, spaces between attributes are just one line. Good job overall.

**Review by eamador1 — COMMENTED** [review]
You did a great job in your coding. Good indentation and spacing.

File: style.css [hunk 3]
```diff
@@ -405,77 +390,127 @@ h2 {
...
 
-.message-label {
-  flex: 1;
+.email::placeholder {
   font-family: 'Poppins', sans-serif;
-  font-size: 10px;
-  margin-left: 5vh;
+  font-size: 16px;
+  padding-left: 10px;
+  color: #1a2236;
```
Comment (sylvester-wamaya) [inline style.css:466]: I kindly suggest you add `padding-top` on the email input so that your design looks like Figma's template.

**Review by sylvester-wamaya — COMMENTED** [review]
Good job on your progress!
Please check comments to make more improvements on your project.

**Pull request merged**